src/components/interaction-designer/block-editors/GenericContactPropertyEditor.vue

```diff
@@ -0,0 +1,205 @@
+<template>
```

Comment:
I don't really understand why we need so much new code here given how similar the config we are writing is to the existing Contact Property Block config. Can you explain? 

I think I'm just going to go ahead and merge because it seems to be working fine but I'd like to create a ticket to find a way to share this functionality.

Reply:
I was trying to reuse code from `components/interaction-designer/block-editors/ContactPropertyEditor.vue`, and realized the component became too much complicated, in fact we have a bunch of conditional rendering which are different for SetContactPropBlock and the other Block.

So I'd like to keep it separated.